crates/manager/src/sources/binary.rs

```diff
@@ -23,7 +23,6 @@ pub fn generate_running_process_status_handle(
 
     let task = async move {
         info!("Starting process execution for {service_name}");
-        let _ = status_tx.send(Status::Running).ok();
```

Comment:
?

Reply:
already the default when you check for the status, sending this twice through the channel trigger the above case and abort the blueprint.

Reply:
ah ok, I didn't test this in registration mode.